src/Representation/Types/StructType.cs

```diff
@@ -25,23 +26,26 @@ public IType GetType(ISpace space, IExpression index)
 
     public IExpression GetOffsetBits(ISpace space, IExpression index)
     {
-        return space.CreateConstant(space.PointerSize, (uint) _offsets[(int) index.GetSingleValue(space)]);
+        return space.CreateConstant(space.PointerSize, (uint) _offsets[(int) index.GetSingleValue(space)].ToBits());
```

Comment:
Could this not be `GetOffsetBytes().ToBits()`

Reply:
It's an expression already so this is the only layer I can do it at. Could do a multiply but probably worse that way.